images/node/README.md

```diff
@@ -35,41 +35,104 @@ It specifies `NODE_PORT=3000` by default.
 <!--compatibility:end-->
 
 <!--body:start-->
-## Usage Example
+## Node Application Example
```

Comment:
[Non-blocking comments]
Appreciate you working to improve our docs :) From a consumer there is quite a lot of info here to digest, and it's not the easiest to follow.

Just a suggestion, but would it be cleaner to show them an example of how to use our image to run an existing node application? i.e heres an example of how to use our image with an existing node application.

We could link to the accademy and mention it for more verbose docs on how to build and run node applications?

Reply:
Thanks @mamccorm, I think we have not started doing this yet, but we were planning to do some segmentation of the readmes through Jamon's Terraform work. @SharpRake can touch base with @jamonation about the recommendations there, but maybe this would prove a solution to have a front and center TL;DR that can live on the READMEs and also the Directory, and then have a longer-form example on Academy. Here is the doc that outlines this: https://docs.google.com/document/d/1QpaYcHLafizSyjoNSahZfybeESPEaWDNbcI4oWqM4nY/edit

Reply:
+1 to what Lisa said!

I think for now we should publish these changes to get them out the door but I won't mark [my issue for the node README](https://github.com/chainguard-dev/edu/issues/1214) as done until I've connected with Jamon and got a plan together about how we can approach this better.